openassessment/celery.py

```diff
@@ -0,0 +1,23 @@
+"""
+Import celery, load its settings from the django settings
+and auto discover tasks in all installed django apps.
+
+Taken from: http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html
+"""
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
+from django.conf import settings
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'proj.settings')
```

Comment:
If we run via `manage.py` this will already be set. No need to override it. Besides `proj.settings` is not a valid settings file.